src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,42 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    static final String NUMBER_REG_EXP = "^[0-9]+$";
+    static final String DELIMITER_REG_EXP = "//(.)\\n(.*)";
+
+    public static int splitAndSum(String inputString) {
+        if (inputString == null || inputString.isEmpty()) {
+            return 0;
+        }
+        String[] splitArray = splitInputString(inputString);
+        return sumNumbers(convertStringArrayToIntArray(splitArray));
+    }
+
+    public static String[] splitInputString(String inputString) {
+        Matcher matcher = Pattern.compile(DELIMITER_REG_EXP).matcher(inputString);
+        if (matcher.find()) {
+            String delimiter = matcher.group(1);
+            return matcher.group(2).split(delimiter);
+        }
+        return inputString.split(",|:");
+    }
+
+    public static int[] convertStringArrayToIntArray(String[] splitArray) {
+        return Arrays.stream(splitArray).mapToInt(StringCalculator::stringToInt).toArray();
```

Comment:
```suggestion
        return Arrays.stream(splitArray)
            .mapToInt(StringCalculator::stringToInt)
            .toArray();
```
stream 관련 메소드는 한 줄에 한 개만 사용하도록 해주세요!

Reply:
```java
return matcher
         .group(NUMBER_GROUP)
         .split(delimiter);
```
stream 관련 메소드만 줄바꿈을 하면 될까요? 아니면 위 코드처럼 다른 메소드들도 줄바꿈을 하는게 좋을까요?

Reply:
일단 Stream에만 적용해도 좋을 것 같습니다 ㅎㅎ